src/dtclient/agent_tenant_info.go

```diff
@@ -7,16 +7,16 @@ import (
 	"github.com/pkg/errors"
 )
 
-type TenantInfo struct {
-	ID                    string
+type AgentTenantInfo struct {
```

Comment:
Maybe a common base with `ActiveGateTenantInfo`, or include one in the other?

BTW `Endpoints` use different types in both cases (`string` vs `[]string`), is it necessary?

```suggestion
type TenantInfo struct {
	UUID                  string
	Token                 string
}

type AgentTenantInfo struct {
	TenantInfo
	Endpoints             []string
	CommunicationEndpoint string
}
```

Reply:
Yes, it's necessary because it reflects how REST interfaces are implemented.